### PR TITLE
Create CODE_OF_CONDUCT.md & LICENSE

### PR DESCRIPTION
**Pull request by dlebauer**
This is a default contributor code of conduct suggested by github. Anything that we should add? Our group COC has UA-specific details, but also a statement of inclusivity - worth reviewing both of these documents  https://osf.io/tzmhp/wiki/Code%20of%20Conduct/

Here is the OLS COC for comparison https://github.com/open-life-science/ols-1/blob/master/CODE_OF_CONDUCT.md

see also https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-a-code-of-conduct/

a good but very detailed reference to link to / how to respond: https://frameshiftconsulting.com/code-of-conduct-book/ https://files.frameshiftconsulting.com/books/cocguide.pdf

consequences from OLS discussion
* Suggesting what behaviour to maintain, short vacation from the community, Suggestions on reading and training to improve behaviour or understanding, , banning in worst cases

### CODE REVIEW

**Comment by Chris-Schnaufer** [comment]
Looking at the other documents, do we want to add something along the lines of: "We believe this Code of Conduct reflects our values and principals. Note that you may be additionally bound by any Code of Conduct from your organization"?

**Review by Chris-Schnaufer — APPROVED** [review]

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, sex characteristics, gender identity and expression,
+level of experience, education, socio-economic status, nationality, personal
+appearance, race, religion, or sexual identity and orientation.
+
```
Comment (dlebauer) [inline CODE_OF_CONDUCT.md:11]:
```suggestion

We believe this Code of Conduct reflects our values and principals. Note that you may be additionally bound by any Code of Conduct from your organization.

```

**Comment by dlebauer** [comment]
@Chris-Schnaufer done https://github.com/AgPipeline/.github/pull/1/files#r378448226

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
```
Comment (dlebauer) [inline CODE_OF_CONDUCT.md:55]: need to review this - who to report to (>1 person) and the current response is vague.

**Review by MagicMilly — APPROVED** [review]

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (Chris-Schnaufer) [reply CODE_OF_CONDUCT.md:55]: Should we use the special email: ua-diag-pipeline? We could also put something in here about contacting UA?

**Comment by KristinaRiemer** [comment]
Other possible items: 
- [ ] Identifying specific people to be part of a code of conduct committee (2 people in immediate group and 2 people outside of group?) 
- [ ] Having anonymous mechanism for reporting CoC violations? 
- [ ] Reporting form (see Appendix 2 in Frameshift book) 

Maybe we should merge this PR now and create a new issue for further work?

**Review by KristinaRiemer — APPROVED** [review]

**Pull request merged**